Mathlib/Algebra/Polynomial/UnitTrinomial.lean

```diff
@@ -377,3 +377,4 @@ theorem irreducible_of_coprime' (hp : IsUnitTrinomial p)
 end IsUnitTrinomial
 
 end Polynomial
+
```

Comment:
Is there a reason why the script is sometimes adding empty lines to the end of files ?

Reply:
I am glad that you asked!  I noticed that an earlier version of the script was careless with ending character, so I fixed it (I think), but some of the earlier modifications were not modified again by the updated script.

I'll fix the endings of the files.